docs/guide.md

```diff
@@ -56,7 +56,11 @@ webdriverio
     .end();
 ```
 
-** 6. Run your test file**
+** 6. Install browser and appropriate driver**
+
+In the example above FireFox is specified as the desired browser, but may not be currently installed on the testing system. The choice is to either install FireFox or download [Third Party Drivers](http://docs.seleniumhq.org/download/#thirdPartyDrivers) and place them in the same directory as the selenium standalone server.
```

Comment:
But you say: _"[...] to either install Firefox or install Third Party Drivers [...]"_ which is not quite right. If you install Firefox at the default path you should be find by running a selenium standalone server. However if you only want to run a driver you need to start the driver server. I just don't get what you mean by this sentence.